source/unified-test-format/unified-test-format.rst

```diff
@@ -755,6 +787,13 @@ The structure of each object is as follows:
   executing `operations`_. If the array is empty, the test runner MUST assert
   that no events were observed on the client (excluding ignored events).
 
+- ``ignoreCommandMonitoringEvents``: Optional array of one or more strings.
```

Comment:
It may be worth expanding this paragraph with an instruction that the commands will most likely be logged but must be filtered out when comparing the list of actual events to this list of expected events. Most drivers will most likely first compare the count of both lists and fail the check if it isn't the same, so this technical detail may be helpful.